public/app/features/alerting/unified/components/rule-list/EvaluationGroupWithRules.tsx

```diff
@@ -0,0 +1,85 @@
+import { size } from 'lodash';
+import React from 'react';
+import { useToggle } from 'react-use';
+
+import { createViewLink } from '../../utils/misc';
+import { hashRulerRule } from '../../utils/rule-id';
+import { isAlertingRule, isAlertingRulerRule, isGrafanaRulerRule, isRecordingRulerRule } from '../../utils/rules';
+
+import { AlertRuleListItem, RecordingRuleListItem } from './AlertRuleListItem';
+import EvaluationGroup from './EvaluationGroup';
+import { EvaluationGroupWithRulesProps } from './RuleList.v2';
```

Comment:
Why is this interface located outside of this file?

Reply:
Oh it's definitely not supposed to be, good catch! – probably a left-over refactoring that didn't get cleaned up.